api/app/Models/SecondTestEntity.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: redjik
+ * Date: 13.07.15
+ * Time: 15:07
+ */
+
+namespace App\Models;
+
+class SecondTestEntity extends BaseModel
+{
+    public $table = 'second_test_entities';
+
+    protected $primaryKey = 'entity_id';
+
+    public $timestamps = false;
+
+    /**
+     * @return array
+     */
+    public function getValidationRules()
```

Comment:
This method should be inherited from the parent class, and not defined here if it is just going to return blank.